readme.md

```diff
@@ -1,7 +1,7 @@
 # Moyo header
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Deobp.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Deobp.github.io/layout_moyo-header/report/html_report/)
 
```

Comment:
Consider adding these links to the PR description

Reply:
Not fixed

Reply:
<img width="706" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/47028393/9233a9fe-5154-45a0-a095-4e5818f376dc">